### PR TITLE
Build python-sqlparse in BREW

### LINKED ISSUE #47: Build python-sqlparse in BREW
BREW task: https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=20703595
BREW build: https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=867019

### PR DESCRIPTION
**Pull request by kholdaway**
## Scratch Build Information
A yum repository for the scratch build of python-sqlparse-0.2.4-6.el7 (task 20703448) is available at:
- http://brew-task-repos.usersys.redhat.com/repos/scratch/kholdawa/python-sqlparse/0.2.4/6.el7/

You can install the rpms locally by putting this .repo file in your /etc/yum.repos.d/ directory:
- http://brew-task-repos.usersys.redhat.com/repos/scratch/kholdawa/python-sqlparse/0.2.4/6.el7/python-sqlparse-0.2.4-6.el7-scratch.repo

RPMs and build logs can be found in the following locations:
- http://brew-task-repos.usersys.redhat.com/repos/scratch/kholdawa/python-sqlparse/0.2.4/6.el7/noarch/

The full list of available rpms is:
- http://brew-task-repos.usersys.redhat.com/repos/scratch/kholdawa/python-sqlparse/0.2.4/6.el7/noarch/python-sqlparse-0.2.4-6.el7.src.rpm
- http://brew-task-repos.usersys.redhat.com/repos/scratch/kholdawa/python-sqlparse/0.2.4/6.el7/noarch/python-sqlparse-0.2.4-6.el7.noarch.rpm

Build output will be available for the next 21 days.

### CODE REVIEW

File: rhel7/rcm_data/deps/sqlparse/notes.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Package python-numpy
```
Comment (abaiken) [inline rhel7/rcm_data/deps/sqlparse/notes.md]: change this name to sqlparse

Comment (kholdaway) [reply rhel7/rcm_data/deps/sqlparse/notes.md]: Done!

File: rhel7/rcm_data/deps/sqlparse/python-sqlparse-fedora.spec [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+%files -n python3-%{shortname}
+%license LICENSE
+%doc AUTHORS CHANGELOG README.rst
+%{python3_sitelib}/*
+
+%{_bindir}/sqlformat
+
+%changelog
+* Sat Feb 02 2019 Fedora Release Engineering <releng@fedoraproject.org> - 0.2.4-5
```
Comment (abaiken) [inline rhel7/rcm_data/deps/sqlparse/python-sqlparse-fedora.spec]: optionally add your section to changelog if you want to

**Comment by abaiken** [comment]
add parser to the rcm requirements

File: rhel7/rcm_data/deps/sqlparse/Makefile [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+	CLONE_BRANCH=$(DEFAULT_BRANCH)
+else
+	CLONE_BRANCH=$(branch)
+endif
+
+
+
+help:
+	@echo "Please use \`make <target>' where <target> is one of:"
+	@echo "  setup               Setup RHEL 7 VM"
```
Comment (abaiken) [inline rhel7/rcm_data/deps/sqlparse/Makefile]: add other targets to make help

Comment (kholdaway) [reply rhel7/rcm_data/deps/sqlparse/Makefile]: Great suggestion.  Done.

**Pull request merged**